tempus_fugit_minecraft/main.py

```diff
@@ -4,6 +4,7 @@
 from pyglet.gl import *
 from tempus_fugit_minecraft.window import Window, WINDOW_WIDTH, WINDOW_HEIGHT
 
+
```

Comment:
Why is this extra line of blank space necessary? One blank line between the imports and the first function should be sufficient right?

Reply:
According to PEP8 formatting style, functions should have 2 newlines before them if they are not in a class. See this link: https://peps.python.org/pep-0008/#blank-lines